python/stack_plots.py

```diff
@@ -239,7 +276,7 @@ def get_files(samples,year):
         years=["2016nonAPV","2016APV","2017","2018"]
     for tyear in years:
         for i,sample in enumerate(samples):
-            if sample=="Y3" or sample=="DY3" or sample=="DYp3" or sample=="B3mL2":
+            if sample=="Y3" or sample=="DY3" or sample=="DYp3" or sample=="B3mL2" or sample=="BFF" or sample=="BFFdbs1p0":
```

Comment:
In the `doAll` and `ScanChain` scripts, it is introduced as a "separate" sample, given that it has a different sample name, for convenience. Since we are using `==` to check the `sample` variable, different conditions are needed. I can use the `Contains()` method of TString, if you prefer to have shorter conditions.

Or did I completely miss the point and you prefer that this does not have a different sample name?

Reply:
Sorry, I pointed you to the wrong line.
Below, there are instances of `in` rather than `==`, where one `BFF` instance should be enough.